stellar/README.md

```diff
@@ -79,61 +80,63 @@ node stellar/deploy-contract.js --contractName axelar_operators --wasmPath ../ax
 Generate TypeScript bindings for the contract
 
 ```bash
-node stellar/generate-bindings.js --wasmPath /path/to/optimized.wasm --contractId [contract address] --outputDir ./stellar/bindings/[contract name]
+node stellar/generate-bindings.js --wasmPath /path/to/optimized.wasm --contractId [contract_address] --outputDir ./stellar/bindings/[contract name]
 ```
 
 ## Contract Interaction
 
 Soroban contracts can be interacted directly via the CLI as well. See the help text for individual contract cmds as follows.
 
 ```bash
-soroban contract invoke --network testnet --id [contract-address] --source-account wallet -- --help
+stellar contract invoke --network testnet --id [contract_address] --source-account wallet -- --help
 ```
 
 ### Gateway
 
 To get help on the gateway commands, run:
-`node stellar/gateway.js --help`
+
+```bash
+node stellar/gateway.js --help
+```
 
 #### Call contract
 
 ```bash
 node stellar/gateway.js call-contract [destination_chain] [dstination_address] [payload]
 
 # Example
-node stellar/gateway.js call-contract ethereum 0x4F4495243837681061C4743b74B3eEdf548D56A5 0x1234
+node stellar/gateway.js call-contract avalanche 0x4F4495243837681061C4743b74B3eEdf548D56A5 0x1234
 ```
 
 ### Submit multisig prover proof
 
 Submit a proof constructed on Amplifier to the Stellar gateway contract.
 
 ```bash
-node stellar/gateway.js submit-proof [multisig-session-id]
+node stellar/gateway.js submit-proof [multisig_session_id]
```

Comment:
```suggestion
node stellar/gateway.js submit-proof [multisig-session-id]
```
we should use consistent casing syntax, since cmds are using this as well. Don't know why Stellar CLI uses `_` instead for args

Reply:
Make sense, updated.